project1/workflows/Testsuite4.robot

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Force Tags    set1
+
+
+*** Test Cases ***
+test case 05
+     [tags]    reg    hotels 
+    log    test5
```

Comment:
duplicate emthod can you please use method in common lib

Reply:
this method has been created for complete suite the existing method is not working